src/client/render.tsx

```diff
@@ -71,9 +85,54 @@ export class CellOutput extends React.Component<ICellOutputProps> {
         // Hack, use same classes as used in VSCode for images (keep things as similar as possible).
         // This is to maintain consistently in displaying images (if we hadn't used HTML).
         // See src/vs/workbench/contrib/notebook/browser/view/output/transforms/richTransform.ts
+        const saveAs = () => {
+            if (this.props.ctx.postMessage) {
+                this.props.ctx.postMessage({
```

Comment:
This might be ok for now. But kinda major downside of this being in a different extension. Isn't this just magic string here + magic string on the jupyter extension side to match up? It's just going to break silently if either is changed.